### DISCUSSION TITLE
Include a list without images?

### DISCUSSION THREAD

**Discussion by moraesfernanda** [greenelab/lab-website-template#90] [Q&A] [answered]
Hello! Is there any component that I can select on a list to have a list without images?
I would like to have a literal list of events, like the "list" example from the Basic Formatting tutorial, but with the events that I included in _data/events.yaml :)

**Comment by vincerubinetti** [comment, marked as answer]
Could you post a snippet of your `events.yaml` file, and a screenshot of what you want the list to look like? I'd like to understand your use case better.

If I'm guessing correctly, there's not really a component to do that. That's because, if you just want a simple text list, it's just as easy to write the list directly on your page, like this:

```markdown
### Some Big Event 1

Event description text text text text text text text
text text text text text text text text 

### Some Big Event 2

Event description text text text text text text text text text text text text text text text 
```

Though, I could see how you might want to keep big lists separate from your page content/text by keeping them in `.yaml` files in `_data`. I could also see this being useful if you were displaying events in multiple places on your site and didn't want to duplicate copy/paste.

**Comment by moraesfernanda** [comment]
That's what I meant indeed.
Thank you very much!

Em qua., 4 de ago. de 2021 às 13:12, Vincent Rubinetti <
***@***.***> escreveu:

> Could you post a snippet of your events.yaml file, and a screenshot of
> what you want the list to look like?
>
> If I'm understanding you correctly, there's not really a component to do
> that. That's because, if you just want a simple text list, it's just as
> easy to write the list directly on your Events page, like this:
>
> ### Some Big Event 1
>
> Event description text text text text text text text
> text text text text text text text text
> ### Some Big Event 2
>
> Event description text text text text text text text text text text text text text text text
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/greenelab/lab-website-template/discussions/90#discussioncomment-1130992>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHBUI5LYOVON67SW5QVKSFLT3FRHRANCNFSM5BQE7WCA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Fernanda Hansen P. de Moraes

***@***.***
Doutoranda em Ciências Médicas | Instituto D'Or de Pesquisa e Ensino
